helm/solr-operator/Chart.yaml

```
@@ -54,13 +54,21 @@ annotations:
   # Add change log for a single release here.
   # Allowed syntax is described at: https://artifacthub.io/docs/topics/annotations/helm/#example
   artifacthub.io/changes: |
+    - kind: fixed
+      description: gen-pkcs12-keystore init container fails if the tls secret contains no ca.crt
```

```suggestion
      description: gen-pkcs12-keystore initContainer now supports 'ca.crt'-less TLS secrets
```

Most of the other 'descriptions' here highlight the correct/improved behavior, so I've suggested a similar change above to stay in line with that.